data_utils/raw_dataset_loader.py

```diff
@@ -32,10 +61,14 @@ def load_raw_data(self, dataset_path: str, config):
         config: shows the target variables information, e.g, location and dimension, in data file
         """
 
-        node_feature_dim = config["node_features"]["dim"]
-        node_feature_col = config["node_features"]["column_index"]
-        graph_feature_dim = config["graph_features"]["dim"]
-        graph_feature_col = config["graph_features"]["column_index"]
+        self.node_position_dim = config["node_positions"]["dim"]
```

Comment:
This feature is already in Main branch. Just realized I am reviewing a draft, will come back later.